MADRaS/envs/data/sim_options_v2.yml

```diff
@@ -0,0 +1,250 @@
+torcs_server_port: 60934
```

Comment:
Can we just remove this file for the time being and add the `*.yml` to `.gitignore` to avoid future pushes of config files.

Reply:
Thanks for pointing out the file `sim_options_v2.yml`. Regarding adding *.yml to .gitignore - I am worried that this would cause us to lose track of the addition or removal of control variables in the sim_options file.

Reply:
Let's keep it for the time being until the development goes on.